test/whl-scale-tests/whl-scale-test.md

```diff
@@ -0,0 +1,255 @@
+
+In setting up the Windos Host Log Scale Test Suite you will need to make sure that you have four configurations that focus on each log type. 
+
+To automatically configure the XML for your Geneva Account we will use `.\Docker-Provider\test\whl-scale-tests\prepare-geneva-xml.ps1.` This script will take the xml files in `.\Docker-Provider\test\whl-scale-tests\geneva-config-files` and configure each with your Geneva Account parameters.
+
+### Using prepare-geneva-xml.ps1
+```powershell
+.\prepare-geneva-xml.ps1 `
+    -GenevaAccountName "<your-geneva-account-name>" `
+    -GenevaLogAccountNamespace "<your-geneva-log-account-namespace>" `
+    -GenevaMetricAccountName "<your-geneva-metric-account-name>" `
+    -AKSClusterMSIObjectId "<your-aks-cluster-msi-object-id>"
+```
+Here are some brief notes on the parameters:
+
+- **GenevaAccountName** - Set this to the name of the Geneva Account you wish to use for scale testing WHL.
+    
+- **GenevaLogAccountNamespace** - Set this to the Geneva Logs Account Namespace you wish to use for scale testing WHL.
+    
+- **GenevaMetricAccountName** - Set this to the Geneva Metrics Account you wish to use for scale testing WHL.
+    
+- **AKSClusterMSIObjectId** - Set this to the MSI assigned to your AKS cluster. You can find this by going to the Azure Portal -> search for your MC_[resource_group]\_[cluster_name]_[region] -> find the [cluster_name]-nodepool -> Get the Object ID.
+
+![Example AKS Cluster MSI with Object ID placement circled](images/example_msi_objid.png)
+
+Example:
+```powershell
+.\prepare-geneva-xml.ps1 `
+    -GenevaAccountName "SkyMERProdLogs" `
+    -GenevaLogAccountNamespace "SkyAlulaDevLogs" `
+    -GenevaMetricAccountName "SkyMERProdMetrics" `
+    -AKSClusterMSIObjectId "d1e9226d-17cb-4799-aaf1-79e113f92a70"
+```
+<br>
+
+Once you are done with the script it will open three tab for you in the following order: Geneva Metrics Machine Accesss, Geneva Logs User Roles and a page where you can upload your newly created XML files.
+
+### Assign AKS Cluster Object ID to Geneva Metrics
+
+1. Go to the newly opened Web Browser Window and select the first new tab:
+![Geneva Metris Settings for Machine Access](images/geneva_metric_machine_access.png)
+
+2. Select "Managed Identities"
+
+3. Setup a new row with your AKS Cluster Object ID, your Subscription's Tenant ID, describe the cluster attach to this item and provide the "MetricPublisher" Role. It should look similar to this
+
+    ![New row item with an example of the Object ID, Tenant ID, cluster name and Metric Publisher as the Role](images/metrics_access_managed_identity.png)
+
+4. Click save towards the top of the page. 
+<br>
+
+### Assign AKS Cluster Object ID to Geneva Logs
+
+1. Go next tab:
+
+2. Go to User Role "MdsAccountAdmin_<your_geneva_logs_account>" -> "KeyVault Certs/Identities" and Click "View/Add"

```

Comment:
Should this be <your_geneva_logs_account>_MaCommunication instead? It doesn't seem to work for me when I use MdsAccountAdmin_<your_geneva_logs_account>

Reply:
Yup that is the correct one

Reply:
Fixed